contracts/contracts/interfaces/l1/ILineaRollup.sol

```diff
@@ -190,9 +190,9 @@ interface ILineaRollup {
    * @notice Emitted when L2 blocks have been finalized on L1.
    * @param startBlockNumber The indexed L2 block number indicating which block the finalization the data starts from.
    * @param endBlockNumber The indexed L2 block number indicating which block the finalization the data ends on.
-   * @param shnarf The shnarf being set as currentFinalizedShnarf in the current finalization.
-   * @param parentStateRootHash The indexed parent L2 state root hash that the current finalization starts from.
-   * @param finalStateRootHash The indexed L2 state root hash that the current finalization ends on.
+   * @param shnarf indexed The shnarf being set as currentFinalizedShnarf in the current finalization.
```

Comment:
Can you put  `The indexed shnarf` instead of `indexed The shnarf` to align the working with the params above?